sdk/typescript/test/e2e/read-transactions.test.ts

```diff
@@ -31,29 +31,29 @@ describe('Transaction Reading API', () => {
   it('Get Transactions', async () => {
     const resp = await toolbox.provider.getTransactionsForAddress(
       toolbox.address(),
-      'Ascending'
+      false
     );
     expect(resp.length).to.greaterThan(0);
 
     const allTransactions = await toolbox.provider.getTransactions(
       'All',
       null,
       10,
-      'Ascending'
+      false
     );
     expect(allTransactions.data.length).to.greaterThan(0);
 
     const resp2 = await toolbox.provider.getTransactions(
       { ToAddress: toolbox.address() },
       null,
       null,
-      'Ascending'
+      false
```

Comment:
I think using union types `ascending` here is much more readable than a boolean - can we change it back?